applications/GeoMechanicsApplication/custom_elements/U_Pw_base_element.hpp

```diff
@@ -173,11 +173,10 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) UPwBaseElement : public Element
 
     virtual std::size_t GetNumberOfDOF() const;
 
-    StressStatePolicy& GetStressStatePolicy() const;
+    StressStatePolicy&                   GetStressStatePolicy() const;
+    [[nodiscard]] virtual DofsVectorType GetDofs() const;
```

Comment:
For me it's not completely clear why the GetDofs function needs to be protected now, does the diff order element use it?

Reply:
Also here I agree with Richard. If `SmallStrainUPwDiffOrderElement` only provides the implementation (which I think it does), but it doesn't call this member itself, `private` should be fine.

Reply:
private function also works.